--- setup.py	2021-09-13 06:59:58.000000000 -0500
+++ setup.py	2021-09-15 08:44:10.840642958 -0500
@@ -35,7 +35,7 @@
 setup(
     name="owlrl",
     packages=["owlrl"],
-    scripts=["scripts/owlrl", "scripts/RDFConvertService"],
+    entry_points={"console_scripts": ["owlrl = owlrl._cli:main"]},
     package_dir={"owlrl": "./owlrl"},
     version=version,
     description="OWL-RL and RDFS based RDF Closure inferencing for Python",
